node/bft/src/helpers/mod.rs

```diff
@@ -15,6 +15,10 @@
 pub mod cache;
 pub use cache::*;
 
+// TODO: rename so it is right next to cache
```

Comment:
The `Cache` on `Gateway` is mostly metrics-oriented at the moment. I don't think it would be unreasonable to consolidate but eventually metrics and caching should be separate (with metrics being exposed with e.g. Grafana). Either way, the Gateway will need some refactoring in future, perhaps consolidating to reduce complexity/fragmentation in the meantime.